src/pages/trader-tools/_validation.js

```diff
@@ -55,7 +54,7 @@ const validation = {
         if (!input) {
             return localize('Point value is required')
         } else if (!validation_regex.number.test(input)) {
-            return localize('Invalid point value input [0 - 9]')
+            return localize('Should be a valid number')
```

Comment:
These functions are alsmost same. Is there anyway to make them integrated? Do you think is it a good idea?

Reply:
for this issue because we haven't got the full requirement per each field, I made everything the same

* might change after QA and the new requirement are given ( so for now , I think it's okay to do it this way temporarily )